blocks/v2-resources/v2-resources.css

```diff
@@ -75,3 +75,75 @@ a.v2-resources__link:hover {
     transform: translateX(-644px);
   }
 }
+
+/* MAGAZINE VARIANT */
+.v2-resources--magazine .v2-resources__heading {
+  font-family: var(--ff-headline-compressed);
+  font-size: var(--heading-compressed-font-size);
+  line-height: var(--heading-compressed-line-height);
+  margin: 0 0 56px;
+  max-width: 70%;
+}
+
+.v2-resources--magazine .button-container {
+  margin-top: 24px;
+}
+
+.v2-resources-container.section--background-with-dots.v2-resources-wrapper__magazine::after {
+  height: 60%;
+  width: 70%;
+  left: -15%;
+  bottom: 45%;
+  scale: 0.5;
+}
+
+@media (min-width: 744px) {
```

Comment:
Just a note: I’ve noticed that 768px is used for tablets in other places. I think we should standardize and ensure every block uses the same breakpoint at some point.